core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/impl/AbstractEvaluationStrategyFactory.java

```diff
@@ -43,4 +46,11 @@ public void setOptimizerPipeline(QueryOptimizerPipeline pipeline) {
 		return Optional.ofNullable(pipeline);
 	}
 
+	public boolean isTrackResultSize() {
+		return trackResultSize;
+	}
+
+	public void setTrackResultSize(boolean trackResultSize) {
+		this.trackResultSize = trackResultSize;
+	}
```

Comment:
This is important one. This is used by the strict evaluation strategy to determine if it should append the tracking iteration.

Reply:
Another option: "isResultSizeTracking" and "setResultSizeTracking"